src/Apps/Order/Routes/Accept/index.tsx

```diff
@@ -0,0 +1,220 @@
+import { Button, Flex, Sans, Serif, Spacer } from "@artsy/palette"
+import { Accept_order } from "__generated__/Accept_order.graphql"
+import { Helper } from "Apps/Order/Components/Helper"
+import { TwoColumnLayout } from "Apps/Order/Components/TwoColumnLayout"
+import { ContextConsumer } from "Artsy"
+import { Mediator } from "Artsy/SystemContext"
+import { RouteConfig, Router } from "found"
+import React, { Component } from "react"
+import { createFragmentContainer, graphql, RelayProp } from "react-relay"
+import { Col, Row } from "Styleguide/Elements"
+import { HorizontalPadding } from "Styleguide/Utils/HorizontalPadding"
+import { Media } from "Utils/Responsive"
+import {
+  counterofferFlowSteps,
+  OrderStepper,
+} from "../../Components/OrderStepper"
+
+import { ShippingSummaryItemFragmentContainer as ShippingSummaryItem } from "Apps/Order/Components/ShippingSummaryItem"
+import { TransactionDetailsSummaryItemFragmentContainer as TransactionDetailsSummaryItem } from "Apps/Order/Components/TransactionDetailsSummaryItem"
+import { CountdownTimer } from "Styleguide/Components/CountdownTimer"
+import { get } from "Utils/get"
+import { ArtworkSummaryItemFragmentContainer as ArtworkSummaryItem } from "../../Components/ArtworkSummaryItem"
+import { CreditCardSummaryItemFragmentContainer as CreditCardSummaryItem } from "../../Components/CreditCardSummaryItem"
+
+interface AcceptProps {
+  mediator: Mediator
+  order: Accept_order
+  relay?: RelayProp
+  router: Router
+  route: RouteConfig
+}
+
+interface AcceptState {
+  isCommittingMutation: boolean
+}
+
+export class Accept extends Component<AcceptProps, AcceptState> {
+  state = {
+    isCommittingMutation: false,
+  }
+
+  onSubmit: () => void = () => {
+    console.log("SUBMITTED!")
+  }
+
+  render() {
+    const { order } = this.props
+    console.log("ORDER", order)
+    const { isCommittingMutation } = this.state
+    const artwork = get(
+      this.props,
+      props => order.lineItems.edges[0].node.artwork
+    )
+
+    return (
+      <>
+        <HorizontalPadding px={[0, 4]}>
+          <Row>
+            <Col>
+              <OrderStepper
+                currentStep="Review"
+                steps={counterofferFlowSteps}
+              />
+            </Col>
+          </Row>
+        </HorizontalPadding>
+        <HorizontalPadding>
+          <TwoColumnLayout
+            Content={
+              <Flex
+                flexDirection="column"
+                style={isCommittingMutation ? { pointerEvents: "none" } : {}}
+              >
+                <Media at="xs">
+                  <ArtworkSummaryItem order={order} />
+                </Media>
+                <Media at="xs">
```

Comment:
Can we put this `Spacer` into the above `<Media at "xs"` component?

Reply:
Moving this spacer into the same query causes the bottom border of `ArtworkSummaryItem` to disappear at xs viewports:

<img width="607" alt="screen shot 2018-12-07 at 12 52 04 pm" src="https://user-images.githubusercontent.com/4432348/49664135-fddecc80-fa1e-11e8-8926-caee7796f8f1.png">

Any ideas for how to get around that?

Reply:
Thanks! That worked.